### ISSUE TITLE
Input file validation for merge-methcounts

### ISSUE THREAD

**Issue by martinjvickers** [smithlabcode/methpipe#123]
**TL:DR** `merge-methcounts` does not check the existence of the input files before starting to run and fails with a segmentation fault if one of the input files does not exist.

This isn't a problem but might be a nice quality of life improvement for the future to check the files exist before running. 

I was just using `merge-methcounts` on a large number of meth files I'd converted from bismark and on one file I received a segfault;

```
srun merge-methcounts -t A_uniq.CHG.meth B_uniq.CHG.meth -o A_vs_B_uniq_CHG.proportion.txt
/home/x250/bin/methpipe/3.4.3/x86_64/bin/merge-methcounts: line 3: 17865 Segmentation fault      singularity exec $DIR/../methpipe-3.4.3.img $(basename "$0") $@
srun: error: node001: task 0: Exited with exit code 139
```

I was a bit confused as the file appeared (at first glance until I realised what was happening) normal. On 
closer inspection I had realised that I'd missed one of the bismark to meth file conversions;

```
tail A_uniq.CHG.meth B_uniq.CHG.meth
==> A_uniq.CHG.meth <==
Pt	140066	+	CHG	0	0
Pt	140068	-	CHG	0	0
Pt	140169	+	CHG	0	0
Pt	140172	-	CHG	0	0
Pt	140189	+	CHG	0	0
Pt	140298	-	CHG	0	1
Pt	140357	+	CHG	0	0
Pt	140359	-	CHG	0	1
Pt	140372	+	CHG	0	0
Pt	140374	-	CHG	0	0
tail: cannot open ‘B_uniq.CHG.meth’ for reading: No such file or directory
```

Obviously it's a silly mistake on my part but it ran for 10 mins and produced a sizeable output file and an error that didn't relate to what the issue was.

Many thanks, methpipe is a really great tool, thank you for the support and maintaining it.

**Comment by mengzhou** [comment]
Thanks for the feedback! We have a check function for other tools, e.g `methcounts`, but `merge-methcounts` is not one of them. I will fix this.
